From 1acce67ef18bc7cff34f8f0d8455dd1e7e38812c Mon Sep 17 00:00:00 2001
From: Christoph Junghans <junghans@lanl.gov>
Date: Wed, 26 Jun 2019 11:57:51 -0600
Subject: [PATCH 1/2] cmake: add soversion to libH5mdCore

---
 src/core/io/writer/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/core/io/writer/CMakeLists.txt b/src/core/io/writer/CMakeLists.txt
index d0256da721..b0ce2c1d4a 100644
--- a/src/core/io/writer/CMakeLists.txt
+++ b/src/core/io/writer/CMakeLists.txt
@@ -7,5 +7,6 @@ target_include_directories(H5mdCore SYSTEM PUBLIC
 
 add_dependencies(H5mdCore EspressoConfig)
 target_include_directories(H5mdCore PRIVATE ${CMAKE_SOURCE_DIR}/src/core ${CMAKE_BINARY_DIR}/src/core)
+set_target_properties(H5mdCore PROPERTIES SOVERSION ${SOVERSION})
 install(TARGETS H5mdCore LIBRARY DESTINATION ${LIBDIR})
 endif()

