--- espresso/src/core/io/mpiio/mpiio.cpp.orig	2019-10-04 12:17:47.025423034 -0600
+++ espresso/src/core/io/mpiio/mpiio.cpp	2019-10-04 12:18:04.708572591 -0600
@@ -448,7 +448,7 @@
       int blen = boff[i + 1] - boff[i];
       auto &bl = local_particles[id[i]]->bl;
       bl.resize(blen);
-      std::copy_n(&bond[boff[i]], blen, bl.begin());
+      if(blen) std::copy_n(&bond[boff[i]], blen, bl.begin());
     }
   }
 
